builds/jazz-build-module/aws-lambda-events-module.groovy

```diff
@@ -137,13 +137,13 @@ def removeS3EventsFromServerless(isEventSchdld){
 
 def checkS3BucketExists(s3BucketName){
   try {
-    sh "aws s3api head-bucket --bucket $s3BucketName --output json"
+    sh "aws s3api head-bucket --bucket $s3BucketName --profile cloud-api --output json"
     echo "Bucket exists and have access"
     return true
   } catch (ex) {//bucket exists but with no access
     def res
     try {
-      res = sh(script: "aws s3api head-bucket --bucket $s3BucketName --output json 2<&1 | grep -c 'Forbidden'", returnStdout: true).trim()
+      res = sh(script: "aws s3api head-bucket --bucket $s3BucketName --profile cloud-api --output json 2<&1 | grep -c 'Forbidden'", returnStdout: true).trim()
```

Comment:
u can skip the variable and return directly from here. no need for if condition below